crates/biome_dependency_graph/src/dependency_graph.rs

```diff
@@ -121,8 +121,12 @@ impl DependencyGraph {
             added_or_updated_paths,
             removed_paths,
         ));
-        let resolver =
-            ResolverGeneric::new_with_cache(resolver_cache.clone(), ResolveOptions::default());
+        let resolve_options = ResolveOptions::default()
+            .with_extension(".jsx")
+            .with_extension(".ts")
+            .with_extension(".tsx")
```

Comment:
`.with_extension()` appends the extension to the list, while the default is `[".js", ".json", ".node"]`. I think `[".js", ".jsx", ".ts", ".tsx", ".json", ".node"]` is much popular order of the resolving priority.